deidentification.R

```diff
@@ -0,0 +1,108 @@
+library(synapser)
+library(synapserutils)
+library(tidyr)
+library(dplyr)
+
+synLogin()
+
+unlink('./dictionaries/', recursive = T, force = T)
+
+# Get dictionaries --------------------------------------------------------
+system('synapse get -r syn52316269 --downloadLocation ./dictionaries/ --manifest suppress')
```

Comment:
This is where you get into R as not an optimal pipelining language.   That said, what are these dictionaries?

Reply:
@thomasyu888 https://www.synapse.org/#!Synapse:syn52316269